pandas/io/pytables.py

```diff
@@ -4140,6 +4145,17 @@ def write(
             # set the table attributes
             table.set_attrs()
 
+            if track_times is not None:
+                from tables import __version__ as tables_version
+                from packaging.version import Version
```

Comment:
this import can be at the top (but use LooseVersion)

Reply:
Moved. LooseVersion used.

Reply:
I had to revert
```
from tables import __version__ as tables_version
```

 back here because of:

```
 Checks on imported code1s
##[error]Process completed with exit code 2.
Run source activate pandas-dev
Check import. No warnings, and blacklist some optional dependencies
err: pandas should not import: tables, numexpr
Check import. No warnings, and blacklist some optional dependencies DONE
##[error]Process completed with exit code 2.
```